test/src/e2e_vm_tests/README.md

```diff
@@ -0,0 +1,76 @@
+# Running end-to-end VM tests
+
+This document assumes you have `fuel-core` running on the default port.
+
+## Running all tests
+
+In a second terminal, `cd` into the `sway` repo and run:
+
+```sh
+cargo run --bin=test
+```
+
+After the test suite runs, you should see:
+
+```console
+Tests passed.
+_n_ tests run (0 skipped)
+```
+
+By default, `cargo` uses the debug build mode which might result in slow
+execution. To speed testing up, you might want to use the release build mode:
+
+```sh
+cargo run --release --bin=test
+```
+
+This may speed the tests up by an order of magnitude.
+
+## Running specific tests
+
+From the `sway` directory run:
+
+```sh
+cargo run --bin=test -- specific_tests_pattern
```

Comment:
I just checked and `cargo run --bin=test specific_tests_pattern` works and `--` isn't required.

cc @sezna 

Doesn't really matter though, approving still.

Reply:
Oh TIL.